pypyr/pipelinerunner.py

```diff
@@ -224,8 +188,22 @@ def load_and_run_pipeline(pipeline_name,
     # try to run a failure-handler from the pipeline, but if the pipeline
     # doesn't exist there is no failure handler that can possibly run so this
     # is very much a fatal stop error.
-    pipeline_definition = get_pipeline_definition(pipeline_name=pipeline_name,
-                                                  working_dir=working_dir)
+    loader_module = pypyr.moduleloader.get_module(loader)
+
+    try:
+        getattr(loader_module, "get_pipeline_definition")
```

Comment:
since you're already doing the getattr here, might as well use the return function reference. . .

`get_pipeline_definition = getattr(loader_module, 'get_pipeline_definition')`

(single quotes since this for machine and not humans, while we're here ;-) )